### PR TITLE
Use stopwatch for ingestjob to make timing a bit cleaner

### PR DESCRIPTION
**Pull request by alerman**

### CODE REVIEW

**Review by apmoriarty — APPROVED** [review]

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java [hunk 1]
```diff
@@ -249,7 +250,11 @@ protected void printUsage() {
 
     @Override
     public int run(String[] args) throws Exception {
-        long setupStart = System.currentTimeMillis();
+
+        long start = System.currentTimeMillis();
```
Comment (FineAndDandy) [inline warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java:254]: never referenced

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java [hunk 2]
```diff
@@ -420,12 +426,18 @@ public int run(String[] args) throws Exception {
...
-        long setupStop = System.currentTimeMillis();
-        log.info("JOB SETUP TIME: " + (setupStop - setupStart) + "ms");
+        sw.stop();
+        log.info(formatTaskInfo(sw.getLastTaskInfo()));
 
+        sw.start("MR Job");
         job.waitForCompletion(true);
+        sw.stop();
+
         long stop = System.currentTimeMillis();
```
Comment (FineAndDandy) [inline warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java:436]: never referenced

**Review by FineAndDandy — CHANGES_REQUESTED** [review]

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java (hunk 1 above)
Comment (alerman) [reply warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java:254]: see call to writeStats on line 532

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java (hunk 2 above)
Comment (alerman) [reply warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java:436]: see call to writeStats on line 532

**Review by avgAGB — APPROVED** [review]

**Review by FineAndDandy — APPROVED** [review]

**Review by hlgp — APPROVED** [review]

**Pull request merged**